README.md

```diff
@@ -11,7 +11,7 @@ Image processing and manipulation in JavaScript.
 
 `$ npm i image-js`
 
-## API
+## [API](image-js.github.io/image-js-typescript/)
```

Comment:
You need to add the `https://`, this link doesn't work. See https://github.com/image-js/image-js-typescript/blob/4b30cd1618fdcf38760b396b3747919bff4612a4/README.md